modules/exoplanets/exoplanetsmodule.cpp

```diff
@@ -354,6 +372,10 @@ void ExoplanetsModule::internalInitialize(const ghoul::Dictionary& dict) {
         _starGlareTexturePath = p.starGlareTexture->string();
     }
 
+    if (p.planetDefaultTexture.has_value()) {
+        _planetDefaultTexturePath = p.planetDefaultTexture->string();
+    }
+
```

Comment:
Not really about your addition, but seems to me these if statements could use the `value_or` function like the first enabled property?

Reply:
No, it cannot.  Note that we do not get the value directly, but use the `string()` operator to get a string version of the `std::filesystem:path`

The `StringProperty` is not compatible with values of `std::filesystem:path` directly, but we want to use that to get the correct verification of the value in the asset. (However, as a side note, I think that we should probably have a specialized property for files, since we use them so often and would want special features in the UI that make them easier to use, for example)